<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PmrWorkspacesWindow</class>
 <widget class="QDockWidget" name="PmrWorkspacesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMR Workspaces</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <action name="actionAuthenticate">
   <property name="icon">
    <iconset resource="../res/PmrWorkspaces_ui.qrc">
     <normaloff>:/oxygen/actions/pmr-unauthenticated.png</normaloff>:/oxygen/actions/pmr-unauthenticated.png</iconset>
   </property>
   <property name="text">
    <string>Log on</string>
   </property>
   <property name="statusTip">
    <string>Log on to PMR</string>
   </property>
  </action>
  <action name="actionUnauthenticate">
   <property name="icon">
    <iconset resource="../res/PmrWorkspaces_ui.qrc">
     <normaloff>:/oxygen/actions/pmr-authenticated.png</normaloff>:/oxygen/actions/pmr-authenticated.png</iconset>
   </property>
   <property name="text">
    <string>Log off</string>
   </property>
   <property name="statusTip">
    <string>Log off PMR</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../res/PmrWorkspaces_ui.qrc">
     <normaloff>:/oxygen/actions/list-add.png</normaloff>:/oxygen/actions/list-add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create a new workspace</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../res/PmrWorkspaces_ui.qrc">
     <normaloff>:/oxygen/actions/help-about.png</normaloff>:/oxygen/actions/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Show information about the selected object</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/PmrWorkspaces_ui.qrc"/>
 </resources>
 <connections/>
</ui>
